packages/OptionButton/src/OptionButton.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from 'react';
+import cx from 'classnames';
+
+import {
+  TextIcon,
+  OptionScaleIcon,
+  MultipleChoiceIcon,
+  LikertIcon,
+  TextIconDisabled,
```

Comment:
for reduce the number of icons and the size of file you can manage the color via css. I put an example [here](https://codesandbox.io/s/icon-color-disabled-reikpt?file=/src/App.js).